src/using-workflow/using-docker-images.md

```diff
@@ -33,13 +33,9 @@ In order to deploy Docker images, they must conform to the following requirement
 Use `deis create` to create an application on the [controller][].
 
     $ mkdir -p /tmp/example-go && cd /tmp/example-go
-    $ deis create
+    $ deis create example-go --no-remote
     Creating application... done, created example-go
 
-!!! note
-    The `deis` client uses the name of the current directory as the
-    default app name.
-
```

Comment:
I'm referring to the other commands below like `deis pull`. For example, try the following:

```
><> mkdir /tmp/foo
><> cd /tmp/foo/
><> deis create example-go --no-remote
Creating Application... done, created example-go
remote available at ssh://git@deis.10.245.1.3.xip.io:2222/example-go.git
><> deis pull deis/example-go
Creating build... Error: 
404 Not Found
detail: Not found.
```

That's why I'm saying that this note is important. Not to `deis create`, but to every other command in the CLI that targets an existing application.